### ISSUE TITLE
unable to find Pocket_graphs 

### ISSUE THREAD

**Issue by divyavani01** [JK-Liu7/AttentionMGT-DTA#5]
while running protein_process.py, I am facing an issue inside the Protein_graph_process(dataset, fold, dir_output, id_train, id_test) function.
I did not find any code which helps to generate pocket_graphs.
can someone help me out to know how are the pocket_graphs generated.......in the file protein_process.py the code is loading pocket graphs and at the below pasted line i am facing error.

protein_graph_train, _ = load_graphs('data/' + dataset + '/processed' + '/pocket_graph/' + str(id) + '.bin')

**Comment by Lucky-Stark-01** [comment]
Yeah, I'm also facing similar problem. I couldn't find the pocket graph .bin files after preprocessing the protein file.
protein_graph_train, _ = load_graphs('data/' + dataset + '/processed' + '/pocket_graph/' + str(id) + '.bin')

**Comment by ravivarmapatturi** [comment]
how to generate pocket_graph bin files ?? I'm facing issue while running protein_process.py file . 


protein_graph_train, _ = load_graphs('data/' + dataset + '/processed' + '/pocket_graph/' + str(id) + '.bin')

**Comment by JK-Liu7** [comment]
Thank you for your interest in our work. The generated ‘pocket_graph’ file has been uploaded.

**Comment by divyavani01** [comment]
**ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (19964,) + inhomogeneous**
 I got above error while running protein_process.py
@JK-Liu7  how to resolve above issue.

**Comment by wang-ff** [comment]
> Thank you for your interest in our work. The generated ‘pocket_graph’ file has been uploaded.

Could you please tell me how to generate pocket_graph bin files, I really want to know the way to generate pocket_graph bin files and ESM_embedding_pocket npy files, thank you very much
@JK-Liu7
